Automattic/jetpack
# AI Assistant: Integrate AI Control component on Jetpack AI Form

Closes #32203 

## Proposed changes:

* Use AI Control component on the Jetpack AI Form extension popover placement
* Add `width` prop in `UI` context data.
* Fix bug on AI Control component disabled state flag
* Add `AI Control` with `full-width`.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Go to editor
* Insert `Form` block.
* Click on `AI Assistant` toolbar icon.
* You should see the new input control component.

#### Demo

https://github.com/Automattic/jetpack/assets/1663717/66ec61fb-9d9a-4ce5-a8b9-76570b946077
